<ui version="4.0" stdsetdef="1" >
  <class>QAxSelect</class>
  <comment>*********************************************************************
**
** Copyright (C) 2022 The Qt Company Ltd.
** Contact: https://www.qt.io/licensing/
**
** This file is part of the ActiveQt framework of the Qt Toolkit.
**
** $QT_BEGIN_LICENSE:COMM$
**
** Commercial License Usage
** Licensees holding valid commercial Qt licenses may use this file in
** accordance with the commercial license agreement provided with the
** Software or, alternatively, in accordance with the terms contained in
** a written agreement between you and The Qt Company. For licensing terms
** and conditions see https://www.qt.io/terms-conditions. For further
** information use the contact form at https://www.qt.io/contact-us.
**
** $QT_END_LICENSE$
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
**
******************************************************************************</comment>
 <class>QAxSelect</class>
 <widget class="QDialog" name="QAxSelect">
  <property name="windowTitle">
   <string>Select ActiveX Control</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="filterLineEdit">
       <property name="placeholderText">
        <string>Filter</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="ActiveXList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="unnamed">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel1">
         <property name="text">
          <string>COM &amp;Object:</string>
         </property>
         <property name="buddy">
          <cstring>ActiveX</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ActiveX"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="unnamed2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel2">
         <property name="text">
          <string>Sandboxing:</string>
         </property>
         <property name="buddy">
          <cstring>SandboxingCombo</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="SandboxingCombo"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
